exercises/high-scores/src/main/java/HighScores.java

```diff
@@ -0,0 +1,21 @@
+class HighScores {
+    public HighScores(int[] highScores) {
```

Comment:
Personally, I think this should take a List since a player's high scores are likely to change over time. Arrays are better if the collection has a fixed size.